contracts/src/models/mancala_game.cairo

```diff
@@ -307,6 +307,26 @@ impl MancalaImpl of MancalaGameTrait {
         self.last_move
     }
 
+    // capture all remaining seeds on the final player side before calling the `set_winner` function
+    // this ensures there is no seed left on the board before setting the winner
+    fn final_capture(ref self: MancalaGame, player_one: GamePlayer, player_two: GamePlayer) {
+        if player_one.is_finished() {
```

Comment:
you'd still need to set the pits where you captured the seeds to 0. right now, you are only adding the value of the current pit to the mancala, but not updating the pits where the seeds where removed to 0.

Reply:
Ah yeah true thanks !